sonaranalyzer-dotnet/src/SonarAnalyzer.Common/Protobuf/AnalyzerReport.proto

```diff
@@ -25,7 +25,7 @@ option csharp_namespace = "SonarAnalyzer.Protobuf";
 option java_package = "org.sonarsource.dotnet.protobuf";
 option java_outer_classname = "SonarAnalyzer";
 
-option optimize_for = LITE_RUNTIME;
+option optimize_for = SPEED;
```

Comment:
Is it related to this change/PR?

Reply:
Nope, except that it was preventing to load the project in Eclipse. that's why I kept it as a separate commit.

Reply:
Do you know if it has any impact or if we can merge without fear?

Reply:
The size of the generated code is a bit bigger. To me it is even safer than the previous flag, since it is the default. LITE_RUNTIME was introduce to target Android, and will be moved to a protoc plugin.